stratum/tools/gnmi/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# CMake build file for Stratum code.
+#
+# Copyright 2022 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+
+add_executable(gnmi_cli
+    gnmi_cli.cc
+    $<TARGET_OBJECTS:stratum_glue_o>
```

Comment:
This links in more support code than you need and makes you dependent on additional protobuf libraries.

I have an experimental branch that generates a new library (stratum_utils) which has a smaller footprint and no dependencies on compiled protobufs. (gnmi_cli itself still requires gnmi_proto)

stratum_utils can also be used with tdi_pipeline_builder.